packages/app-builder-lib/templates/nsis/portable.nsi

```diff
@@ -1,15 +1,12 @@
 !include "common.nsh"
 !include "extractAppPackage.nsh"
 
-# https://github.com/electron-userland/electron-builder/issues/3972#issuecomment-505171582
 CRCCheck off
 WindowIcon Off
 AutoCloseWindow True
 RequestExecutionLevel ${REQUEST_EXECUTION_LEVEL}
 
 Function .onInit
-  SetSilent silent
```

Comment:
In silent mode, the splash image will NOT be shown

Reply:
Without specified splash should be silent, so, here we need some condition.

Reply:
oh, I missed this

Reply:
Ohh, that's correct. I'm thinking something like this:
``` 
!ifndef SPLASH_IMAGE
  SetSilent silent
!endif
```